php/Terminus/Models/Environment.php

```diff
@@ -876,6 +869,63 @@ public function workflow($workflow) {
   }
 
   /**
+   * Sifts through the backups data and adds needful properties
+   *
+   * @param [stdClass] $raw_backups Backups data object from Request data
+   * @return [array] $backups
+   */
+  private function annotateBackups($raw_backups) {
```

Comment:
Rather than inject new attributes into the data, I would prefer that we create a `backup` model and instead make these methods we ask of the model. 

This corresponds to my preference for trying to keep separate server-persisted data and data that is solely used for formatting or output.